.circleci/config.yml

```diff
@@ -1026,7 +1036,7 @@ jobs:
       - *setup_env
       - run:
           name: "Test"
-          command: spot_run_tests canary uniswap_trade_on_l1_from_l2.test.ts docker-compose.yml
+          command: spot_run_test_script ./scripts/run_tests canary uniswap_trade_on_l1_from_l2.test.ts canary docker-compose.yml
```

Comment:
Do we need separate steps for this? Does it make sense to build and test on the same job?